tools/rest_api.rb

```diff
@@ -87,8 +87,8 @@ def run
       params    = {}
 
       opts = Trollop.options do
-        version "#{API_CMD} #{VERSION} - ManageIQ REST API Access Script"
-        banner <<-EOS
+        version("#{API_CMD} #{VERSION} - ManageIQ REST API Access Script")
+        banner(<<-EOS)
```

Comment:
I think putting HEREDOC statements in parens looks awful and is quiet confusing.  It also doesn't conform from what I assumes one of it's roots is, bash/shell programming:  http://tldp.org/LDP/abs/html/here-docs.html